### PR TITLE
Forms - Advanced Options

### PR DESCRIPTION
**Pull request by sjcox-rh**
Includes design section, overview section and images for advanced options story.

### CODE REVIEW

**Comment by serenamarie125** [comment]
In the "Collapsed Section" the image shows 2 numbers, but there are 3 items noted.  Please update the image.

**Comment by serenamarie125** [comment]
Shouldn't the Please note text be listed under the label itself on #2 in the Collapsed Section @sjcox-rh ?  Are we going to recommend an action such as "Show ..." and suggest that it remains the same or changes to "Hide" when expanded ?

File: pattern-library/forms-and-controls/advanced-options/design/design.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+![Image of expanded section](img/advancedoptions_example2.png)
+
+1. **Caret Icon:** When expanded, the caret icon will be facing down.
+
+2. **Additional fields:** Will be shown under the divider line when section is expanded.
+
+## Keyboard Accessibility
+
+* Users should be able to use the keyboard to navigate to the control (i.e., it should gain focus) and press enter to activate the control.
+* Tooltips are also a requirement for accessibility.
```
Comment (andresgalante) [inline pattern-library/forms-and-controls/advanced-options/design/design.md]: @sjcox-rh What do you mean by tooltips?

**Comment by LHinson** [comment]
@serenamarie125 terminology for this is a different story. https://patternfly.atlassian.net/browse/PTNFLY-1831

@sjcox-rh

- +1 to move the "please note" section under item #2. 
- You can consolidate callout #1 and #2 so that you don't have to update the image.

**Comment by sjcox-rh** [comment]
@serenamarie125 @LHinson Thanks for the feedback.  Took a look at the issues in question and agree with you both.  Thanks for catching both of those @serenamarie125! I will update the .md file to reflect the changes.  

@andresgalante, when I update, do I have to submit a new PR or will it be another commit under this same PR?

File: pattern-library/forms-and-controls/advanced-options/design/design.md (hunk 1 above)
Comment (sjcox-rh) [inline pattern-library/forms-and-controls/advanced-options/design/design.md]: @andresgalante Tooltips that appear when you hover over an element and a little box appears to tell you what the element is, or more info on the element.

**Comment by andresgalante** [comment]
@sjcox-rh A PR is a direct link to your branch. Once you update your local files and push them to origin, this PR will be updated.

If you have any question or need help please ping me.

**Comment by sjcox-rh** [comment]
Perfect, thanks @andresgalante for clarifying!  WIll update today :]

**Comment by andresgalante** [comment]
@sjcox-rh this one got messy 😄 let me know if I can help

**Comment by sjcox-rh** [comment]
@andresgalante yeah, still not sure what happen.  Should I just delete it and resubmit a new PR?

**Pull request closed without merging**